From 6b46ecee7ace3558574c0db51bebd1d3f15b0d66 Mon Sep 17 00:00:00 2001
From: Oliver Kurth <okurth@gmail.com>
Date: Tue, 12 Dec 2023 14:17:25 -0800
Subject: [PATCH] remove stdc++fs

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 402e3e80..24a83be8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -308,7 +308,6 @@ else()
     util
     resolv
     atomic
-    stdc++fs
   )
 endif()
 
-- 
2.34.1

